### DISCUSSION TITLE
How to make Tasmota continue working without WiFi

### DISCUSSION THREAD

**Discussion by adamdyga** [arendst/Tasmota#10167] [Q&A] [answered]
I have a Tasmota-controlled smart socket in an area (outside) where WiFi connectivity is very poor and Tasmota often loses the Wifi connection. Lack of Wifi connection in my case is not a problem, because I don't need permanent control over the device. I have configured the timers to do the job and actually WiFi is not required for normal operation. Unfortunately when Tasmota loses the Wifi connection, it enters AP mode and it seems the timers stop working (or am I missing something?). Can this be changed, so that Tasmota continues working (especially the timers) even if there is no WiFi? 
I see the `WifiConfig` command (https://tasmota.github.io/docs/Commands/#wi-fi) allows several options here. Should it be `Wifi 5`? If so, will it possible to reset the device to "factory settings" if the original WiFi network is not available anymore?

**Comment by Frogmore42** [comment, marked as answer]
I use this:
```
4 = retry other AP without rebooting (default)
```
Just set another AP (doesn't have to exist).  It should just switch between them.  If that doesn't work, you can try 5.  You should still be able to reset back to factory settings, but I usually just reflash.

I have been really impressed with the current core version (and Tasmota).  I have two devices that are a long way from the router.  The RSSI goes below 10% (-95dB) sometimes.  As long as it is above about 10% they stay connected.  Looking at the heap size, they probably are doing some retries, but nothing that concerns me.

**Comment by catalintabirca** [comment]
Some functions need correct time and date. You got to have a way to update time in case of reset/cold start without wifi (ex: Ds3231)
